dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -5,49 +5,48 @@
 #NLS_MESSAGEFORMAT_NONE
 
 authFilter=Authentication Filter
-authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers to determine whether or not the HTTP request is selected for the authentication.
+authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication.
 
 webApp=Web Application
-webApp.desc=Specifies the web application name.
+webApp.desc=Specifies the web application name. For example, <webApp id="sample" name="app1|app2" matchType="notContain"/>
 
 requestUrl=Request URL
-requestUrl.desc=Specifies the request URL.
+requestUrl.desc=Specifies the HTTP request URL. For example, <requestUrl id="sample" urlPattern="/account/checking|/account/saving" matchType="contains" />
 
 urlPattern=URL pattern
-urlPattern.desc=Specifies the URL pattern.
-
-remoteAddress=Remote Host
-remoteAddress.desc=Specifies the remote host.
+urlPattern.desc=Specifies the URL pattern. The "*", when used as a wildcard, it is not supported.
+remoteAddress=Remote Address
+remoteAddress.desc=Specifies the remote host TCP/IP address of the client application that sent the HTTP request. For example, <remoteAddress id="sample" ip="100.100.100.99" matchType="greaterThan"/>
 
 ip=IP address
-ip.desc=Specifies the IP address.
+ip.desc=Specifies the remote host TCP/IP address. 
 
 host=Host
-host.desc=Specifies the host name.
+host.desc=Specifies the host name. For example, <host id="sample" name="host.mycomp.com" matchType="contains"/>
 
 name=Name
 name.desc=Specifies the name.
 
 value=Value
-value.desc=Specifies the value.
-valueOptional.desc=The value attribute specifies the HTTP request header value. If the value is not specified, then matching is done with the name attribute, not the value attribute.
+value.desc=Specifies the HTTP request header value.
+valueOptional.desc=The value attribute specifies the request header value. If the value is not specified, then matching is done with the name attribute, not the value attribute. For example, <requestHeader id="sample" name="email" matchType="contains"/>
```

Comment:
Suggested update:
"The value attribute specifies the value of the request header. If the value is not specified, then the name attribute is used for matching, for example, `<requestHeader id="sample" name="email" matchType="contains"/>`."